lib/l10n/app_en.arb

```diff
@@ -143,10 +143,40 @@
 
   "pageHelpTitle": "Help",
   "pageHelpContactUs": "Contact Us",
-  "pageHelpWhatIsMotisQuestion": "What can this app do?",
-  "pageHelpWhatIsMotisAnswer": "With the MOTIS Ride App, you can do a lot of useful stuff!",
-  "pageHelpWhereIsAccessibilityQuestion": "Where are the accessibility settings?",
-  "pageHelpWhereIsAccessibilityAnswer": "This app supports a multitude of accessibility settings offered by the OS. Go to your OS's settings to enable Screen Reader or large fonts!",
+  "pageHelpEmailSubject": "Inquiry by User-ID [{id}]",
+  "@pageHelpEmailSubject": {
+    "placeholders": {
+      "id": {}
+    }
+  },
+  "pageHelpQuestion1": "What is MOTIS?",
+  "pageHelpAnswer1": "MOTIS encompasses many tools to support intermodal routing. Its goal is to help plan journeys that include public transport, bike sharing, ride sharing and more modes of transportation.",
+  "pageHelpQuestion2": "How much do rides cost? How can I pay?",
+  "pageHelpAnswer2": "We designed the payment process to fully rely on user agreements. Both the cost and payment method of a ride can be negotiated in person or in our in-app chat. You will see a price suggestion for most rides as a starting point.",
+  "pageHelpQuestion3": "We agreed on a price, but now my rider isn't paying it. What can I do?",
+  "pageHelpAnswer3": "We're sorry for the inconvenience. Unfortunately, the only thing we can offer you is the ability to report users in that case. Tap the \"Report user \" button on their profile and make sure to select the reason \"Did not pay the agreed amount\".",
```

Comment:
Don't say we can't do anything. Just tell them to please report them and send us the evidence via mail or so. What we could say is that we can't take any legal actions since that will have to come from the users side

Reply:
Yeah but that isn't an answer to the question is it? I think the question users are going to ask themselves in that situtation is if there is something we can do to force riders to pay their fair price, and I think it is fair to the users to admit that there isn't. I specifically avoided a phrase like "We can't do anything", though, and I think this text succeeds in conveying the things users CAN do more than what we can't do.